pom.xml

```diff
@@ -220,6 +220,7 @@
                                     <directory>${project.basedir}/src/main/resources/</directory>
                                     <includes>
                                         <include>start-iguana.sh</include>
+                                        <include>iguana.owx</include>
```

Comment:
Please also add the ontology version to `iguana --version`. Not sure if we defined the option yet. If not, please add.